.ci/packaging.groovy

```diff
@@ -257,8 +267,24 @@ def release(){
   }
 }
 
+def runE2ETests(){
+  def suites = '' // empty value represents all suites in the E2E tests
+
+  catchError(buildResult: 'UNSTABLE', message: 'Unable to run e2e tests', stageResult: 'FAILURE') {
+    def suitesSet = e2eTestSuites.toSet()
```

Comment:
`toSet()` is allowed by Jenkins sandbox:
- https://github.com/jenkinsci/script-security-plugin/blob/master/src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist